src/components/Filter/index.js

```diff
@@ -61,30 +62,32 @@ export default class Filter extends Component {
   };
 
   render() {
-    const { filterRenderer, data, dataKey, filterIconRenderer } = this.props;
+    const { filterRenderer, data, dataKey, filterTrigger } = this.props;
     return (
-      <div
-        className="Sticky-React-Table--Header-Column-Filter"
-        style={filtersStyles}
-      >
-        <div
-          className="Sticky-React-Table--Header-Column-Filter-Icon"
-          onClick={this.handleIconClick}
-          ref={this.handleIconRef}
-          style={filterIconStyles}
+      <Fragment>
```

Comment:
`Fragment` is not necessary as you are already rendering everything inside a `span`